code/modules/lighting/lighting_system.dm

```diff
@@ -338,10 +338,7 @@
 		if(newalpha >= LIGHTING_DARKEST_VISIBLE_ALPHA)
 			newalpha = 255
 		if(lighting_object.alpha != newalpha)
-			if(instantly)
```

Comment:
Probably not but idk. You can also do a search within DM or through grep or notepad++ or whatever external tool you prefer.